src/calendar/js/calendar.js

```diff
@@ -102,6 +102,22 @@ Y.Calendar = Y.extend(Calendar, Y.CalendarBase, {
         event.preventDefault();
     },
 
+    /**
+     * Prevent button focus on tab navigation, enable on arrow navigation
+     * @method _setTabIndex
+     * @param node {Node} The parent node
+     * @param tabIndexValue {Number} Corresponding to the value to be updated in the tabindex
+     * @protected
+     */
+    _setTabIndex : function (node, tabIndexValue) {
+        var buttonChild = node.querySelector('button');
+        buttonChild.setAttribute('tabindex', tabIndexValue);
+
+        if(tabIndexValue === 0) {
+            buttonChild.focus();
```

Comment:
What caused focus to be added to the gridcell before? I expected to see us manually set focus to the button, but I thought we would also be removing code that manually set focus to the cell.

Reply:
answered [here](https://github.com/liferay/yui3/pull/44#discussion_r1742671204)